packages/edit-site/src/components/global-styles/size-control/index.js

```diff
@@ -45,7 +49,7 @@ function SizeControl( props ) {
 	};
 
 	return (
-		<BaseControl { ...baseControlProps }>
+		<BaseControl { ...baseControlProps } __nextHasNoMarginBottom>
```

Comment:
## Testing instructions

In Global Styles, go to Typography ▸ Font size presets ▸ (whatever option). Enable the Fluid typography and Custom fluid values toggles.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/4770859e-a7d6-40f2-abd6-859f3885449a" alt="Font size presets, before" width="274">|<img src="https://github.com/user-attachments/assets/4d633e7a-bc25-4132-b4ba-4274ceaeb441" alt="Font size presets, after" width="274">|

I normalized the spacing.

I extracted out a labeling bug as a separate issue: #64406.